internal/cli/kraft/kraft.go

```diff
@@ -79,6 +80,9 @@ func NewCmd() *cobra.Command {
 	cmd.AddCommand(set.NewCmd())
 	cmd.AddCommand(unset.NewCmd())
 
+	cmd.AddGroup(&cobra.Group{ID: "lib", Title: "MICROLIBRARY COMMANDS"})
```

Comment:
Let's call it `PROJECT LIBRARY COMMANDS`

Reply:
this for all  of them then :D

Reply:
Done 👍🏽